Sources/PackageDescription/PackageDependency.swift

```diff
@@ -221,44 +229,162 @@ extension Package.Dependency {
     @available(_PackageDescription, obsoleted: 5.2)
     public static func package(
-        path: String
+        url: String,
+        _ requirement: Package.Dependency.Requirement
     ) -> Package.Dependency {
-        return .init(name: nil, url: path, requirement: .localPackageItem)
+        precondition(!requirement.isLocalPackage, "Use `.package(path:)` API to declare a local package dependency")
+        return .init(name: nil, url: url, requirement: requirement)
     }
 
-    /// Adds a package dependency to a local package on the filesystem.
-    ///
-    /// The Swift Package Manager uses the package dependency as-is
-    /// and doesn't perform any source control access. Local package dependencies
-    /// are especially useful during development of a new package or when working
-    /// on multiple tightly coupled packages.
+    /// Adds a remote package dependency with a given version requirement.
     ///
-    /// - Parameters
-    ///   - name: The name of the Swift package or `nil` to deduce the name from path.
-    ///   - path: The local path to the package.
+    /// - Parameters:
+    ///     - name: The name of the package, or `nil` to deduce it from the URL.
+    ///     - url: The valid Git URL of the package.
+    ///     - requirement: A dependency requirement. See static methods on `Package.Dependency.Requirement` for available options.
     @available(_PackageDescription, introduced: 5.2)
     public static func package(
         name: String? = nil,
-        path: String
+        url: String,
+        _ requirement: Package.Dependency.Requirement
     ) -> Package.Dependency {
-        return .init(name: name, url: path, requirement: .localPackageItem)
+        precondition(!requirement.isLocalPackage, "Use `.package(path:)` API to declare a local package dependency")
+        return .init(name: name, url: url, requirement: requirement)
+    }
+}
+
+// MARK: - registry
+
+extension Package.Dependency {
+    /// Adds a package dependency that uses the version requirement, starting with the given minimum version,
+    /// going up to the next major version.
+    ///
+    /// This is the recommended way to specify a remote package dependency.
+    /// It allows you to specify the minimum version you require, allows updates that include bug fixes
+    /// and backward-compatible feature updates, but requires you to explicitly update to a new major version of the dependency.
+    /// This approach provides the maximum flexibility on which version to use,
+    /// while making sure you don't update to a version with breaking changes,
+    /// and helps to prevent conflicts in your dependency graph.
+    ///
+    /// The following example allows the Swift Package Manager to select a version
+    /// like a  `1.2.3`, `1.2.4`, or `1.3.0`, but not `2.0.0`.
+    ///
+    ///    .package(identity: "scope/name", from: "1.2.3"),
```

Comment:
In the proposal, we defined package identifiers as `scope.name`. I'd be happy to amend that to `scope/name` while we have https://github.com/apple/swift-evolution/pull/1410 open. Otherwise, this should be changed to: 


```suggestion
    ///    .package(identity: "scope.name", from: "1.2.3"),
```

(Same for lines [313](https://github.com/apple/swift-package-manager/pull/3635/files#diff-232fa4d4517851a3c5d1580128275cbffe4b0064c5849379a6637617dc056eacR313), [332](https://github.com/apple/swift-package-manager/pull/3635/files#diff-232fa4d4517851a3c5d1580128275cbffe4b0064c5849379a6637617dc056eacR332), and [351](https://github.com/apple/swift-package-manager/pull/3635/files#diff-232fa4d4517851a3c5d1580128275cbffe4b0064c5849379a6637617dc056eacR351))